accessibility-checker-engine/src/v2/dom/DOMMapper.ts

```diff
@@ -36,7 +36,12 @@ export class DOMMapper extends CommonMapper {
         return retVal;
     }
 
-    getBounds(node: Node) : Bounds {
+    /**
+     * get scaled bounds for screenshot etc. adjusted for devicePixelRatio and scroll
+     * @param node 
+     * @returns 
+     */
+    getAdjustedBounds(node: Node) : Bounds {
```

Comment:
To avoid breaking existing implementations, this function should not be renamed, but rather the new function should have a different name. This will change the bounds calculation in the report, which, if someone is using for correlating with a screenshot, would now break.
```suggestion
    getBounds(node: Node) : Bounds {
```